### PR TITLE
Add a basic SMF manifest for SunOS platforms

### PR DESCRIPTION
**Pull request by tuxillo**
Hi,

A basic SMF file which might come handy for people use SunOS platforms like SmartOS, OmniOS, etc.

Regards,
Antonio Huete

### CODE REVIEW

**Review by lafriks — APPROVED** [review]

**Review by tonivj5 — APPROVED** [review]

**Comment by sjorge** [comment]
Nice, that saves me the effort of writing one. This one works perfectly on SmartOS.

**Comment by tuxillo** [comment]
@sjorge I've also made a port in pkgsrc WIP :-)

**Review by sapk — APPROVED** [review]

File: contrib/init/sunos/gitea.xml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+		<property_group name="application" type="application"></property_group>
+		<property_group name="startd" type="framework">
+		  <propval name="duration" type="astring" value="child"/>
+		  <propval name="ignore_error" type="astring" value="core,signal"/>
+		</property_group>
+
+		<template>
+			<common_name>
+			  <loctext xml:lang="C">Compact self-hosted GIT service</loctext>
```
Comment (techknowlogick) [inline contrib/init/sunos/gitea.xml]: Can you use "A painless, self-hosted Git service" instead? It is the tagline for Gitea

**Review by techknowlogick — CHANGES_REQUESTED** [review]

**Comment by sjorge** [comment]
@tuxillo nice, I could not find it on the joyent/pkgsrc-wip repo though?

**Comment by lafriks** [comment]
@techknowlogick need your approval

**Comment by tuxillo** [comment]
@sjorge it's in pkgsrc-wip (https://github.com/NetBSD/pkgsrc-wip/commit/6f84130e7cb5ec214241a861b2a450db57613a78)

**Review by techknowlogick — APPROVED** [review]

**Pull request merged**